dev/TabView/TabView.cpp

```diff
@@ -428,3 +435,28 @@ winrt::DependencyObject TabView::ContainerFromIndex(int index)
     }
     return nullptr;
 }
+
+void TabView::OnTabViewKeyUp(const winrt::IInspectable&, const winrt::KeyRoutedEventArgs& args)
```

Comment:
Someone on a [Discord](https://discordapp.com/channels/372137812037730304/580484470877061120/599047267009429514) that I just joined pointed out that Ctrl+F4 seems to be handled on Down for browsers -- we should probably do that here too (Alt+F4 is on down as well).

Reply:
That means if you hold down Ctrl+Tab, it'll keep firing and switching tabs - is that desirable?

---
In reply to: [303190443](https://github.com/microsoft/microsoft-ui-xaml/pull/1047#discussion_r303190443) [](ancestors = 303190443)